### ISSUE TITLE
Unable to Reload Extension on Remote Server

### ISSUE THREAD

**Issue by vs74** [cdr/sshcode#170]
Hi, I am facing a problem when using sshcode ; when try to reload an installed extension it doesn't work. For example for my case, I am trying to reload `Python `extension for my project but unfortunately, it doesn't work.
However, this seems to not an issue when working with the code server.

Here is the screenshot of the Pytho n extension I am trying to relaod.
<img width="353" alt="Screenshot 2020-03-26 at 5 23 45 PM" src="https://user-images.githubusercontent.com/31419210/77670522-9ab3d900-6f86-11ea-950b-d00904307541.png">

**SYSTEM SPECIFICATIONS**
**OS: Mac OS
Remote Server : Ubuntu 16.04**

**Comment by Merith-TK** [comment]
I would reccomend installing the "Reload" extension, it basically adds a button in the bottom right that reloads. Or you can Refresh the page (CTRL/CMD+R?), does the same thing

try that

**Comment by vs74** [comment]
Actually, I tried these options suggested by you a couple of times but they didn't work. But now for some reason, it did work, I had to restart my remote server and local machine. Sorry for opening a naive issue.

**Comment by Merith-TK** [comment]
No worries, if you can share logs it may be able to help the devs figure out why it happened for you in the first place.

**Comment by Merith-TK** [comment]
Any luck?>

**Comment by vs74** [comment]
Yes, it's working good. Sorry, I don't have the logs for this issue ( I didn't save the logs as my problem got solved afterwards). But if in future I get the same issue, I will share the logs. Thanks for helping out.

**Comment by vs74** [comment]
I again encountered this problem. Whenever, I used the don't use the` skipsync` flag, while launching vscode, I face this problem. But if I use skipsync flag, the problem get's resolved. 

Unfortunately, I don't see any logs of this issue on my local machine. Is there a way, I could see the logs and share them here ?

**Comment by Merith-TK** [comment]
Can you give the ENTIRE terminal output from sshcode?

**Comment by stahldom** [comment]
I'm also unable to reload extensions. I uninstalled them. When attempting to reinstall them they are unable to be installed because they no longer work on the version used by sshcode. Code-server works. When will sshcode update?

**Comment by vs74** [comment]
@stahldom , I guess this issue is related to #135.
